tests/test_browser.py

```diff
@@ -3629,6 +3629,24 @@ def test_pthread_c11_threads(self):
                expected='0',
                args=['-g4', '-std=gnu11', '-xc', '-s', 'USE_PTHREADS', '-s', 'PROXY_TO_PTHREAD', '-s', 'TOTAL_MEMORY=64mb'])
 
+  @requires_threads
+  def test_pthread_pool_size_strict(self):
+    # Check that it doesn't fail with sufficient number of threads in the pool.
+    self.btest(path_from_root('tests', 'pthread', 'test_pthread_c11_threads.c'),
+               expected='0',
+               args=['-g2', '-xc', '-std=gnu11', '-pthread', '-s', 'PTHREAD_POOL_SIZE=4', '-s', 'PTHREAD_POOL_SIZE_STRICT=2', '-s', 'TOTAL_MEMORY=64mb'])
+    # Check that it fails instead of deadlocking on insufficient number of threads in the pool.
+    self.btest(path_from_root('tests', 'pthread', 'test_pthread_c11_threads.c'),
+               expected='1',
+               args=['-g2', '-xc', '-std=gnu11', '-pthread', '-s', 'PTHREAD_POOL_SIZE=3', '-s', 'PTHREAD_POOL_SIZE_STRICT=2', '-s', 'TOTAL_MEMORY=64mb'])
```

Comment:
Is `-xc` needed?    The idea is that it should be automatic based on the filename.  Is that not working for btest?

Reply:
For whatever reason, no. I initially copied those params from the original test, then thought "huh, that's weird, surely -xc is not needed", removed it, it failed to compile, and had to add it back.